CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff

```diff
@@ -868,7 +868,7 @@
         <note/>
       </trans-unit>
       <trans-unit id="No override" xml:space="preserve">
-        <source>No override</source>
+        <source>No override</source><target>За замовчуванням</target>
```

Comment:
"By default"

Relatedly, countries and languages are still in English. Maybe, there are bundled-into-OS translations for those? (I'm not an iOS developer)

Reply:
Those are all sourced from The Movie Database; I can look into seeing if they have localized versions, but I don't _think_ so.